install_config/cluster_metrics.adoc

```diff
@@ -600,25 +600,38 @@ To avoid this issue, use certificates which are configured to be acceptable by
 your browser.
 ====
 
-ifdef::openshift-origin[]
 == Accessing Hawkular Metrics Directly
 
 To access and manage metrics more directly, use the Hawkular Metrics API.
 
+[NOTE]
+====
+When accessing Hawkular Metrics via the API you will only be able to perform
+reads, writing metrics has been disabled by default.  If you wish for
```

Comment:
consider reworking to be:
perform reads. Writing metrics is disabled by default.